src/UI/Art/SpriteArtist/SpriteLibrary.java

```diff
@@ -605,7 +605,8 @@ public static BufferedImage getCoOverlay(Commander co, boolean leftSide)
     if( !coOverlays.containsKey(co) )
     {
       // If we don't already have this overlay, go load and store it.
-      Sprite overlay = new Sprite(SpriteLibrary.loadSpriteSheetFile("res/ui/co_overlay.png"), CommanderOverlayArtist.OVERLAY_WIDTH, CommanderOverlayArtist.OVERLAY_HEIGHT);
+      final BufferedImage sprite = SpriteLibrary.loadSpriteSheetFile("res/ui/co_overlay.png");
+      Sprite overlay = new Sprite(sprite, sprite.getWidth()/2, sprite.getHeight());
```

Comment:
Why width/2?

Reply:
The sprite has the left and right overlays next to each other, so this tells the `Sprite` to treat them as sub-frames.